server/src/internalClusterTest/java/org/elasticsearch/monitor/metrics/NodeIndexingMetricsIT.java

```diff
@@ -0,0 +1,164 @@
+                "es.indexing.coordinating_operations.size"
+            );
+            assertThat(coordinatingOperationsSize.getLong(), greaterThan(0L));
+
+            var coordinatingOperationsTotal = getRecordedMetric(
+                plugin::getLongAsyncCounterMeasurement,
+                "es.indexing.coordinating_operations.total"
+            );
+            assertThat(coordinatingOperationsTotal.getLong(), equalTo((long) docsCount));
+
+            var coordinatingOperationsCurrentSize = getRecordedMetric(
+                plugin::getLongGaugeMeasurement,
+                "es.indexing.coordinating_operations.current.size"
+            );
+            assertThat(coordinatingOperationsCurrentSize.getLong(), equalTo(0L));
+
+            var coordinatingOperationsCurrentTotal = getRecordedMetric(
+                plugin::getLongGaugeMeasurement,
+                "es.indexing.coordinating_operations.current.total"
+            );
+            assertThat(coordinatingOperationsCurrentTotal.getLong(), equalTo(0L));
+
+            var coordinatingOperationsRejectionsTotal = getRecordedMetric(
+                plugin::getLongAsyncCounterMeasurement,
+                "es.indexing.coordinating_operations.rejections.total"
+            );
+            assertThat(coordinatingOperationsRejectionsTotal.getLong(), equalTo(0L));
+
+            var coordinatingOperationsRejectionsRatio = getRecordedMetric(
+                plugin::getDoubleGaugeMeasurement,
+                "es.indexing.coordinating_operations.rejections.ratio"
+            );
+            assertThat(coordinatingOperationsRejectionsRatio.getDouble(), equalTo(0.0));
+
+            var primaryOperationsSize = getRecordedMetric(plugin::getLongAsyncCounterMeasurement, "es.indexing.primary_operations.size");
+            assertThat(primaryOperationsSize.getLong(), greaterThan(0L));
+
+            var primaryOperationsTotal = getRecordedMetric(plugin::getLongAsyncCounterMeasurement, "es.indexing.primary_operations.total");
+            assertThat(primaryOperationsTotal.getLong(), equalTo((long) docsCount));
+
+            var primaryOperationsCurrentSize = getRecordedMetric(
+                plugin::getLongGaugeMeasurement,
+                "es.indexing.primary_operations.current.size"
+            );
+            assertThat(primaryOperationsCurrentSize.getLong(), equalTo(0L));
+
+            var primaryOperationsCurrentTotal = getRecordedMetric(
+                plugin::getLongGaugeMeasurement,
+                "es.indexing.primary_operations.current.total"
+            );
+            assertThat(primaryOperationsCurrentTotal.getLong(), equalTo(0L));
+
+            var primaryOperationsRejectionsTotal = getRecordedMetric(
+                plugin::getLongAsyncCounterMeasurement,
+                "es.indexing.primary_operations.rejections.total"
+            );
+            assertThat(primaryOperationsRejectionsTotal.getLong(), equalTo(0L));
```

Comment:
Can we force some rejections to see this counter growing? Same for the coordination rejections

Reply:
added two more tests